src/gluonts/model/deepar/_network.py

```diff
@@ -264,6 +265,51 @@ def unroll_encoder(
 
 
 class DeepARTrainingNetwork(DeepARNetwork):
+    def distribution(
+        self,
+        feat_static_cat: Tensor,
+        past_time_feat: Tensor,
+        past_target: Tensor,
+        past_observed_values: Tensor,
+        future_time_feat: Tensor,
+        future_target: Tensor,
+        future_observed_values: Tensor,
+    ) -> Distribution:
+        """
+        Returns the distribution of the model on the range of past_target and future_target.
+        This can be used in anomaly detection, see for instance examples/anomaly_detection.py.
```

Comment:
I would add a blank line between these two: the first line should be a quick description of the method, while further details should be included in separate paragraphs.

Reply:
Sure, done.